src/app/public/modules/page-summary/page-summary.component.ts

```diff
@@ -11,6 +11,9 @@ import { Subscription } from 'rxjs/Subscription';
   providers: [SkyPageSummaryAdapterService]
 })
 export class SkyPageSummaryComponent implements OnDestroy, AfterViewInit {
+  @ViewChild('keyInfoContainerEl')
+  public keyInfoContainerEl: ElementRef;
+  public hasKeyInfo: boolean;
```

Comment:
Could we set this to `false` by default?

Reply:
done